Lib/pathlib.py

```diff
@@ -1303,6 +1305,15 @@ def lstat(self):
             self._raise_closed()
         return self._accessor.lstat(self)
 
+    def link(self, target):
+        """
+        Create a hard link pointing to a path named target.
+        If the path is a directory, use mkdir() instead.
```

Comment:
I'm not sure what this mean? `mkdir` is not the same thing as creating a link...

Reply:
I just removed that statement.